pybaseball/statcast_fielding.py

```diff
@@ -0,0 +1,58 @@
+import io
+from typing import Optional, Union
+
+import pandas as pd
+import requests
+
+from . import cache
+from .utils import norm_positions
+
+@cache.df_cache()
+def statcast_outs_above_average(year: int, posn: str, min_att: Union[int, str] = "q") -> pd.DataFrame:
```

Comment:
Isn't 'pos' a more typical abbreviation? (would apply for usages throughout)

Reply:
Yeah I guess it is...I think I went with `posn` to differentiate from `positive`, but will change that